fixes#579 make a better fundraiser by add standalone above bar

bhagyashree980 opened the pull request:
Added a standalone fundraiser banner at the top of the page ,independent of the  navbar
Integrated the banner's content into banner.html.
Conditionally included the banner using the show_fundraiser flag in _config
Updated base.html to include the banner at the top

https://github.com/user-attachments/assets/6a6ad675-7bd1-4e26-a2a6-d7ef820029ea

![WhatsApp Image 2024-12-16 at 11 02 39 PM](https://github.com/user-attachments/assets/05d08bf2-f25a-4059-ae49-4d6291ec1812)

bhagyashree980 commented:
@pikurasa @quozl I’ve gone through everything and checked all the updates thoroughly. I also closed one pull request. I have made  changes to banner.html since that file is handled by its respective owner.

I’ve worked on this for two days, and while I know there might still be some silly mistakes, I’ve tried my best. Please take a moment to review it when you can. Your feedback would be greatly appreciated!

quozl commented on _includes/banner.css:
```diff
@@ -0,0 +1,27 @@
+    margin-left: 10px;
+    font-size: 0.9rem;
+  }
+  
```
Missing newline at end of file.

bhagyashree980 commented:
Hi Quozl,
I’ve made the changes as you suggested:
Removed the unnecessary merge commit by rebasing the branch onto master.
Updated the commit message to clearly describe the changes, focusing on the problem, solution, and the improvements made.
Thank you for pointing out these issues. Let me know if there’s anything else to refine or improve.

quozl commented:
Thanks, great work on the commit messages, however the changes made (see https://github.com/sugarlabs/www/pull/595/files) don't seem right now.  Perhaps you might check the changes and confirm?  The branch also has conflicts to be resolved.

bhagyashree980 commented:
Hii @quozl, Thank you for the feedback. I’ve reviewed the pull request, and it seems that some of the changes included were not made by me. My contribution specifically focused on:
Adding a standalone fundraiser banner above the navigation bar.
Updating the commit message to better describe the changes made.
It looks like other changes might have been introduced from a merge or conflict resolution that included updates from another branch. I’m not sure how those changes were added. Could you guide me on the best way to clean this up and ensure only my intended changes are included?

quozl commented:
Yes, there is certainly a strangeness about the commits on this branch proposed for the pull request.

A pull request in GitHub arises from a branch in git.

## This time

Rewrite your branch commits so that the branch contains only what you want us to merge.  Then `git push --force` will update the pull request.

I'm assuming you have `git remote -v` showing up `sugarlabs` for the remote pointing to our repository, and `origin` for the remote pointing to your own repository.  If this is not the case, substitute `sugarlabs` below accordingly.

You can at any time look at what the pull request will contain by using `git log -p sugarlabs/master`, which does a diff against our master branch and shows you what you have committed.

There are several ways to rewrite commits, of which two are probably the most likely to be successful in this scenario;

### Destroy everything and start again

* `git fetch` from the sugarlabs repository,
* `git reset --hard sugarlabs/master`,
* reapply your changes,
* use `git log -p sugarlabs/master` and _carefully_ review everything that is to be in the pull request,
* `git push --force` to your repository on GitHub, which will update the pull request.

### Change specific commits

* `git rebase -i` to select the commit to edit or reword,
* broadly follow the instructions shown by `git rebase`,
* use `git log -p sugarlabs/master` and _carefully_ review everything that is to be in the pull request,
* `git push --force` to your repository on GitHub, which will update the pull request.

## Next time

You've used your own _master_ branch instead of a topic branch.  As consequence you've been hindered by merging back our  _master_ branch and you've prevented us from contributing easily to your branch.

We recommend using a topic branch, which means at the point you are about to make filesystem changes you type `git checkout -b TOPIC` where TOPIC is something you invent that indicates very broadly what you are doing.

https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md has our general workflow recommendations.

quozl commented:
Looks like the problem you were trying to fix has been solved.  Please re-open if you are solving other problems.
